integration-tests/mock-components/updated-effect.js

```diff
@@ -0,0 +1,13 @@
+const { registerHtml, useObservable, useEffect } = require('../../src/tram-one')
+const html = registerHtml()
+
+module.exports = ({ initialCount }) => {
+	const [counter] = useObservable({ count: Number(initialCount) })
+	const increment = () => { counter.count += 1 }
+	useEffect(() => {
+		document.title = `The count is ${counter.count}`
+		return () => { document.title = `The count was ${counter.count}` }
+	})
+
+	return html`<button data-testid="updatable-button" onclick=${increment}>${counter.count}</button>`
```

Comment:
a large chunk of this is just in the attrs, so I don't know if it would be smaller to wrap this. More often I do that if the child is another element. In this case, it's just text.